tools/docker/compose.md

```diff
@@ -8,36 +8,43 @@ section: tools
 
 [docker-compose](https://docs.docker.com/compose/) is a tool to orchestrate docker containers. No more 4 lines long `docker run` commands, just one YAML file which describes your whole setup.
 
-In this example we'll go through a simple development environment setup:
 
-* first step will be to start a [Django](https://www.djangoproject.com/) project
-* we will bindmount sources inside a docker container (app will change as we code)
-* our choice of database will be PostgreSQL (which we link to the container with sources)
-* the whole development environment will be managed by docker-compose
+## Installation
 
-We will name our project `awesome_web`. So let's start by creating directory for it:
+The package is named `docker-compose`, you can install it easily:
 
 ```
-$ mkdir awesome_web
-$ cd awesome_web
+# dnf install docker-compose
 ```
 
-Whole database will live in subdirectory `db` (postgres user has to own it -- docker still lacks user namespaces):
 
-```
-$ mkdir db
-# chown 26:26 db
-```
+# Docker Compose Tutorial
+
+In this example we'll go through a simple development environment setup:
+
+1. start a [Django](https://www.djangoproject.com/) project
+2. mount sources inside a docker container (application will change as we code)
+3. our choice of database will be PostgreSQL (which we link to the container with sources)
+4. the whole development environment will be managed by docker-compose
+
 
 ## Prerequisites
 
+
+### Docker
+
+This guide expects that you have docker engine installed, configured and running (if your user is not able to reach docker, prefix all `docker` commands with `sudo `; for more information see section docker of this site).
```

Comment:
We might discourage people from using sudo (get back to `#` notation) since new users are not administrators by default. It's a no brainer to still use it in commands, because we would change them all in one commit (if we decide to do so), but referencing this in text seems to be a little problematic. So if it's not much to ask, can you rather mark the commands with `#`?).